infrastructure/police_db/Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM postgres:9.6
+COPY pg.conf /etc/postgresql/postgresql.conf
+CMD ["-c", "config_file=/etc/postgresql/postgresql.conf"]
 
 ## PostGIS activation
 RUN apt-get -y update \
     && apt-get -y  install wget \
     && wget --quiet -O - http://apt.postgresql.org/pub/repos/apt/ACCC4CF8.asc | apt-key add - \
     && apt-get -y update \
-    && apt-get -y install postgresql-9.6-postgis-2.3 postgis postgresql-9.6-pgrouting
-
+    && apt-get -y install postgresql-9.6-postgis-2.3 postgis postgresql-9.6-pgrouting \ 
```

Comment:
You only need the first `apt-get -y install` here. Dockerfile best practices also call for alphabetical ordering of the package names.